package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "scripts": {
+    "format": "prettier --single-quote --trailing-comma=es5 --write '{create-react-native-app,react-native-scripts}/**/*.js'"
+  },
+  "devDependencies": {
+    "prettier": "^0.21.0"
```

Comment:
Need to add the appropriate installation and pre-PR prettying to the contributing guide.